image/tarexport/save.go

```diff
@@ -327,15 +328,17 @@ func (s *saveSession) saveImage(id image.ID) (map[layer.DiffID]distribution.Desc
 	if err := os.WriteFile(configFile, img.RawJSON(), 0o644); err != nil {
 		return nil, err
 	}
-	if err := system.Chtimes(configFile, img.Created, img.Created); err != nil {
-		return nil, err
+	if img.Created != nil {
+		if err := system.Chtimes(configFile, *img.Created, *img.Created); err != nil {
```

Comment:
Should we consider resetting this one to `time.Unix(0, 0)` if `img.Created == nil` to make things more reproducible?

Reply:
I'm not sure about that - that feels wrong to me?  I'm happy to adjust though, and don't feel strongly about it if I'm alone in thinking that'd be weird for things that don't have a `Created` value at all. :sweat_smile: